*** /etc/ImageMagick-6/policy.xml	Thu Jun 10 07:52:45 2021
--- policy.xml	Thu Sep 30 09:06:04 2021
***************
*** 87,96 ****
--- 87,98 ----
    <!-- in order to avoid to get image with password text -->
    <policy domain="path" rights="none" pattern="@*"/>
    <!-- disable ghostscript format types -->
+   <!-- Security issues were fixed in ghostscript version 9.24.
    <policy domain="coder" rights="none" pattern="PS" />
    <policy domain="coder" rights="none" pattern="PS2" />
    <policy domain="coder" rights="none" pattern="PS3" />
    <policy domain="coder" rights="none" pattern="EPS" />
    <policy domain="coder" rights="none" pattern="PDF" />
    <policy domain="coder" rights="none" pattern="XPS" />
+   -->
  </policymap>
